# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../slmodem/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- slmodem-2.9.10/drivers/Makefile	2003-12-21 22:15:54.000000000 +0100
+++ slmodem-2.9.10-fixed/drivers/Makefile	2005-04-25 17:24:34.000000000 +0200
@@ -72,17 +72,14 @@
 install: uninstall-old
 	install -D -m 644 slamr.o $(module-dir)/slamr.o
 	install -D -m 644 slusb.o $(module-dir)/slusb.o
-	cp /etc/modules.conf /etc/modules.conf.slamr && \
-	    echo 'alias char-major-212 slamr' >> /etc/modules.conf && \
-	    echo 'alias char-major-213 slusb' >> /etc/modules.conf 
-	/sbin/depmod -a
+
 uninstall:
 	/sbin/modprobe -r slamr slusb
 	cp /etc/modules.conf /etc/modules.conf.slamr && \
 	   egrep -ve 'alias char-major-212 slamr|alias char-major-213 slusb' /etc/modules.conf.slamr > /etc/modules.conf
 	$(RM) $(module-dir)/slamr.o
 	$(RM) $(module-dir)/slusb.o
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 uninstall-old:
 	$(RM) $(module-dir)/slmdm.o \
               $(module-dir)/slfax.o \
@@ -102,13 +99,13 @@
 install:
 	install -D -m 644 slamr.ko $(module-dir)/slamr.ko
 	install -D -m 644 slusb.ko $(module-dir)/slusb.ko
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 uninstall:
 	modprobe -r slamr ; echo -n
 	modprobe -r slusb ; echo -n
 	$(RM) $(module-dir)/slamr.ko
 	$(RM) $(module-dir)/slusb.ko
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 endif
 
 endif
